.travis.yml

```diff
@@ -72,11 +72,8 @@ script:
 # Deploy debian packages to bintray
 deploy:
     provider: script
-    script: beaver bintray upload -N pkg/deb/*.deb
+    script: beaver bintray upload -N deb/*.deb
```

Comment:
Yeah, the commit which moves the packing scripts to the `pkg` is on the 5.x.x. MIssed that when reviewing after the rebase on v5.5.x